src/main/java/bio/terra/cli/serialization/userfacing/UFGroup.java

```diff
@@ -21,18 +21,28 @@
 public class UFGroup {
   public final String name;
   public final String email;
+  public final Integer numMembers;
   public final List<GroupPolicy> currentUserPolicies;
 
   public UFGroup(Group internalObj) {
     this.name = internalObj.getName();
     this.email = internalObj.getEmail();
     this.currentUserPolicies = internalObj.getCurrentUserPolicies();
+
+    Integer numMembersNotFinal;
+    try {
+      numMembersNotFinal = internalObj.getMembers().size();
+    } catch (Exception ex) {
```

Comment:
Just out of interest, under what circumstances does this throw?  What is the logical difference between `numMembers` being `null` vs zero?

Reply:
This throws if the user does not have permission to list the group members. Only group admins have this permissions, so non-admins can see/describe the group, but they can't see how many people are in it. The text display value will be `(unknown)` if `numMembers` is null. Will add a comment.